src/main/java/build/buildfarm/worker/shard/ShardWorkerContext.java

```diff
@@ -281,12 +290,63 @@ public QueuedOperation getQueuedOperation(QueueEntry queueEntry)
     return ProtoUtils.parseQueuedOperation(queuedOperationBlob, queueEntry);
   }
 
+  // FIXME make OwnedClaim with owner
+  // how will this play out with persistent workers, should we have one per user?
```

Comment:
maybe we just don't make persistent workers work with multiple exec owner feature...

Reply:
Roughly I agree - change justification is that executions should be isolated and not able to access peer address spaces. I think persistent workers is probably going to fly in the face of this. The problem with this feature and persistent workers is the life of a daemon beyond the end of the report result stage. Needs more thought overall.